main.css

```diff
@@ -0,0 +1,771 @@
+    font-size: 40px;
+}
+
+.drive-with__driver{
+    width: 250px;
+    height: 250px;
+}
+
+.drive-with__slider {
+    display: flex;
+    justify-content: center;
+    align-items:flex-end;
+    gap: 60px;
+}
+
+.drive-with__points {
+    display:flex;
+    flex-direction: row;
+    gap: 30px;
+}
+
+.drive-with__arrow {
+    width: 70px;
+    height: 70px;
+}
+
+/* Ride with Lyft */
+
+.ride {
+    display:flex;
+    flex-direction: column;
+    gap: 100px;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    padding: 100px 0px 100px 0px;
+    background-color: #FFFFFF;
+}
+
+.ride__header{
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    text-align: center;
+}
+
+.ride__list{
+    display: flex;
+    flex-direction: column;
+    gap: 60px;
+    text-align: left;
+    padding-left: 30px;
+}
+
+.ride__point{
+    display:flex;
+    flex-direction: row;
+    gap: 30px;
+}
+
+.ride span {
+    font-size: 40px;
+    font-weight: 600;
+    width: 100%;
+}
+
+.ride h2 {
+    font-size: 70px;
+    font-weight: 600;
+    width: 100%;
+}
+
+.ride p {
+    font-size: 30px;
+    font-weight: 300;
+    width: 100%;
+}
+
+.ride__list strong {
+    font-size: 35px;
+    font-weight: 600;
+    width: 100%;
+}
+
+.ride__icon{
+    width: 50px;
+    height: 50px;
+}
+
+.ride__screenshot{
+    height: 1100px;
+    width: 800px
+}
+
+.ride__getride {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    gap: 50px;
+}
+
+.ride__buttons {
+    line-height: 2.5;
+    padding: 0px 70px 0px 70px;
+    font-size: 2.5em;
+    text-align: center;
+    color: #FFFFFF;
+    text-shadow: 1px 1px 1px #000;
+    border-radius: 200px;
+    width: 200px;
+    height: 120px;
+    background-color: #523be4;
+    font-weight: 600;
+}
+
+.ride__pink{
+    display:flex;
+    flex-direction: column;
+    gap: 50px;
+    align-items: center;
+    text-align: center;
+    width: 90%
+}
+
+.ride__pink-image{
+    display: flex;
+    width: 300px;
+    height: 100px;
+}
+
+.ride__pink-link{
+    color:#000
+}
+
+/*choose ride */
+
+.ride__Ride-Bike-Scoot{
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    gap: 60px;
+    padding: 100px 0px 100px 0px;
+}
+
+.ride__slider-container{
+    display:flex;
+    flex-direction: row;
+    gap: 140px
+}
+
+.ride__slider-arrows{
+    width: 100px;
+    height: 100px;
+}
+
+.ride__cards p{
+    font-size: 30px;
+    font-style: italic;
+    width: 100%;
+    text-align: center;
+    color: grey;
+    width: 80%
+}
+
+.ride__cards {
+    text-align: center;
+    color: grey;
+    width: 90%;
+    display:flex;
+    align-items: center;
+    justify-content: center;
+}
+
+/* Estimation pink section */
+
+.estimation{
+    display:flex;
+    flex-direction: column;
+    text-align: left;
+    background-color:#DCDCFF;
```

Comment:
space is missing between rule name and rule value, make sure you fix it everywhere